eng/packages/http-client-csharp-mgmt/generator/TestProjects/Local/Mgmt-TypeSpec/src/Generated/FooData.Serialization.cs

```diff
@@ -92,11 +92,6 @@ internal static FooData DeserializeFooData(JsonElement element, ModelReaderWrite
                     id = new ResourceIdentifier(prop.Value.GetString());
                     continue;
                 }
-                if (prop.NameEquals("name"u8))
```

Comment:
This doesn't seem right.

Reply:
Or I guess this will be fixed once we bump the MTG version to latest?

Reply:
Same reason as https://github.com/Azure/azure-sdk-for-net/pull/49499#discussion_r2055261980

Reply:
[name](https://github.com/Azure/typespec-azure/blob/363736edf7b75ead39ac72282af0b08a2228760e/packages/typespec-azure-resource-manager/lib/models.tsp#L33-L34) is a `@path` parameter 
In [MTG.MrwSerializationTypeDefinition](https://github.com/microsoft/typespec/blob/054baa995d6f7a4391d7d5a637331c54445e97c9/packages/http-client-csharp/generator/Microsoft.TypeSpec.Generator.ClientModel/src/Providers/MrwSerializationTypeDefinition.cs#L799-L802), we are skipping non-body parameters.